daffodil-runtime1/src/main/scala/org/apache/daffodil/processors/DaffodilParseXMLReader.scala

```diff
@@ -144,7 +142,13 @@ class DaffodilParseXMLReader(dp: DataProcessor) extends DFDL.DaffodilParseXMLRea
   }
 
   def parse(isdis: InputSourceDataInputStream): Unit = {
-    val sio = createSAXInfosetOutputter(this)
+    val sio = createSAXInfosetOutputter(this,
+      saxNamespaceFeatureValue,
+      saxNamespacePrefixesFeatureValue)
+    // validate that the features are not false/false
+    if (!sio.namespacesFeature && !sio.namespacePrefixesFeature) {
```

Comment:
Should we do this check before creating the sax infoset outputter? This way we avoid creating a sax infoset outputter to just error and have it be garbage collected.